config/routes.rb

```diff
@@ -63,6 +63,7 @@
     get "/", to: "dashboard#index"
     patch "/", to: "dashboard#update"
     get "/items", to: "dashboard#items"
+    get "/orders/:id", to: "orders#show"
```

Comment:
So this is the same as `get "/merchant/orders/:id", to: "merchant:orders#show"`... and that's what you Slacked about...? Which controller to send it too? 
👍